### PR TITLE
adding partial seed file

### PR DESCRIPTION
**Pull request by AngelRobish**

### CODE REVIEW

File: script/seed.js [hunk 1]
```diff
@@ -1,58 +1,56 @@
...
+  description: 'Here’s our second time working with hickory! This gorgeous bowl has been an eye catcher from the moment we poured it, it’s beauty is just out of this world!',
+  material: 'Hickory',
+  epoxyColor: 'blue',
+  price: 200.00,
+  category: 'Bowl',
+  image: 'https://scontent-mia3-1.xx.fbcdn.net/v/t39.30808-6/276123626_158077886599088_4538128186652999978_n.jpg?_nc_cat=104&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=N0tLt3VXmlEAX9KWbmJ&_nc_ht=scontent-mia3-1.xx&oh=00_AT9sX3xrEBJ8b8LcArmdNUkEs3z7vHDEfJFcAb386oHKuA&oe=6264C172',
+  customizable: true
+}, {
+  name: 'Jewelry box',
+  description: 'Get your own custom stained box, the perfect gift idea for any event your attending!Our first time working with hickory, and it definitely won’t be our last! We hope you love this small beautiful bowl, perfect for whatever you need, as much as we do!',
```
Comment (sommerh) [inline script/seed.js:15]: Small typo - "any event you're attending" It also seems that Hickory bowl description was accidentally pasted onto the end of this

File: script/seed.js [hunk 2]
```diff
@@ -1,58 +1,56 @@
...
- The `seed` function is concerned only with modifying the database.
-*/
-async function runSeed() {
-  console.log('seeding...')
+async function seed () {
   try {
-    await seed()
+    await db.sync({ force: true })
+
+    await Promise.all(products.map(product => {
```
Comment (sommerh) [inline script/seed.js:37]: I am not sure if this will change anything - but maybe use a forEach here since we aren't using whatever is returned from map

Comment (CaitlinPSherman) [reply script/seed.js:37]: we followed seed files in other projects for this

**Comment by sommerh** [comment]
Everything looks good

**Review by sommerh — APPROVED** [review]

**Pull request merged**